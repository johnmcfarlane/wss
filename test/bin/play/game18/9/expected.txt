 39  8,8  - INEZ
 38  8,4  | ZANIES
 37  8,4  | ZAIRES
 35  6,4  - SAZEN
 34  6,4  - SIZAR
 34  6,4  - SIZER
 33  6,4  - SIZE
 32  6,4  - SEZ
 32  8,4  | XENIAS
 30  6,2  - EZINE
 29  6,5  - KNEZI
 28  6,5  - KNEZ
 28  6,4  - SIXER
 27  6,4  - SAXE
 26  6,2  - EXINE
 26  3,1  | NIL
 26  6,4  - SAX
 26  6,4  - SEX
 26  6,4  - SIX
 25  9,5  | RAZEE
 25  9,5  | ZAIRE
 25  5,2  - ZEIN
 24  5,2  - ZEN
 23  3,1  | AIL
 23  8,5  | ZAXES
 23  5,2  - ZEE
 23  5,2  - ZER
 23  7,8  | ZERO
 22  9,7  | ZEE
 21  7,8  - AINE
 21  8,8  - AREN
 21  8,8  - IRAN
 21  7,8  - NARE
 21  7,8  - RANE
 21  7,8  - RINE
 20  4,5  | ALIENOR
 20  3,11 - AX
 20  8,4  | AZINES
 20  6,5  - KNIAZ
 20  4,9  | NOX
 20  6,4  - SINZER
 19  7,8  - AIRE
 19  6,5  - KAREZ
 19  4,6  | LIENOR
 19  1,7  - REDIA
 19  6,4  - SENZA
 19  8,5  | ZEINS
 18  8,8  - ANE
 18  7,1  | AX
 18  7,1  | EX
 18  6,5  - KAZI
 18  1,7  - ZED
 17  9,5  | AZINE
 17  9,5  | EZINE
 17  8,5  | NAZIS
 17  9,6  | RAZEED
 16 10,6  | ANEW
 16  8,8  - ARE
 16  8,8  - ERA
 16  8,8  - IRA
 16  8,8  - IRE
 16  8,5  | IZARS
 16 10,6  | NARW
 16  1,11 - NEAR
 16  8,5  | RAZES
 16  7,8  | RENO
 15  4,2  | ACERLI
 15  7,8  - AIN
 15  4,3  | CANLI
 15  9,5  | EXINE
 15  2,6  | FANES
 15  2,6  | FINES
 15  7,8  - NAE
 15  7,8  - NAR
 15  9,6  | NAZE
 15  7,8  - NEA
 15  8,5  | NIXES
 15  7,8  - RAN
 15  7,8  - REN
 15  7,8  - RIN
 15  4,1  | ZAC
 14  7,8  | AERO
 14  5,5  - AKIN
 14 10,6  | AREW
 14  4,3  | CARLE
 14  4,3  | CARLI
 14  4,3  | CRILE
 14  6,2  - EARNIE
 14  5,4  - ISZ
 14  6,5  - KEX
 14  9,8  - NEA
 14  9,8  - RAN
 14  8,5  | RAXES
 14  9,6  | RAZE
 14  9,8  - REN
 14  6,4  - SIREX
 14  8,7  | ZAS
 14  9,8  | ZEED
 14 11,5  | ZIARA
 13  7,8  - AER
 13  7,8  - AIR
 13  5,5  - AKER
 13  5,5  - AKRE
 13  8,6  | AXES
 13  8,6  | AXIS
 13  1,7  - EADI
 13  7,8  - EAR
 13  7,8  - EIR
 13  6,2  - EZRA
 13 11,6  | EZRA
 13  2,6  | FARES
 13  2,6  | FERES
 13  2,6  | FIRES
 13  5,6  | MAX
 13  5,6  | MIX
 13  9,6  | NIXE
 13  7,8  - REA
 13  7,8  - RIA
 13  7,8  - RIE
 13  6,4  - SEAX
 13 11,5  | XENIA
 13 11,6  | ZIRA
 12  5,5  - AKE
 12  8,8  - AN
 12  3,7  - DIA
 12  3,7  - DIE
 12  8,8  - EN
 12  6,2  - ERIAN
 12  6,2  - ERNIE
 12  8,8  - IN
 12  6,5  - KIERAN
 12  6,5  - KRINA
 12  7,8  - NA
 12  7,8  - NE
 12  1,7  | RAJ
 12  9,8  - REA
 12 10,9  | WRO
 12  8,7  | XES
 12  8,7  | XIS
 12  7,7  | ZEE
 11 10,8  - AN
 11  8,10 - AN
 11  9,7  | AXE
 11 10,8  - EN
 11  8,10 - EN
 11  8,10 - IN
 11  6,5  - KAIN
 11  6,5  - KAREN
 11  6,5  - KARN
 11  6,5  - KEARN
 11  6,5  - KENAI
 11  6,5  - KERN
 11  6,5  - KIRN
 11  6,5  - KRAN
 11  9,8  - NA
 11  9,8  - NE
 11  1,11 - NEA
 11  8,4  | RAINES
 11 11,8  | ZA
 10  7,8  - AE
 10  8,8  - AE
 10  7,8  - AI
 10  8,5  | AIRNS
 10  8,8  - AR
 10  7,7  | AXE
 10  4,3  | CARL
 10  4,3  | CEIL
 10  4,3  | CIRL
 10  2,11 - EAR
 10  9,4  | EARNIE
 10  8,5  | EARNS
 10  8,8  - ER
 10  6,2  - EXR
 10  4,1  | INCA
 10  4,1  | INCR
 10  8,7  | INSET
 10  6,5  - KANE
 10  6,5  - KINA
 10  6,5  - KINE
 10  6,5  - KNAR
 10  6,5  - KRNA
 10  4,1  | NACE
 10  1,7  - NED
 10  4,1  | NICE
 10  8,5  | RAINS
 10  7,8  - RE
 10  8,5  | REINS
 10  6,4  - SANER
 10  6,4  - SINAE
  9  9,8  - AE
  9  9,6  | AENEID
  9  1,7  - AID
  9  9,5  | AINEE
  9  8,6  | AINS
  9  9,5  | ARNEE
  9  9,5  | ERNIE
  9  8,6  | ERNS
  9  6,2  - EX
  9  6,5  - KARI
  9  6,5  - KEIR
  9  6,5  - KIER
  9  5,2  | NAA
  9  5,2  | NAE
  9  9,6  | NEREID
  9  4,1  | RACE
  9  1,7  - RAD
  9  9,5  | RANEE
  9  9,8  - RE
  9  1,7  - RED
  9  4,1  | RICE
  9  1,7  - RID
  9  8,6  | RINS
  9  6,4  - SANE
  9  6,4  - SENA
  9  6,4  - SERIAN
  9  6,4  - SINA
  9  6,4  - SINE
  8  4,2  | ACE
  8  2,7  - AD
  8  3,11 - AN
  8  8,7  | ANS
  8  9,7  | AREND
  8  2,7  - ED
  8  9,7  | ENEID
  8  8,7  | ENS
  8  2,6  | FA
  8  2,6  | FE
  8  2,6  | FI
  8  4,2  | ICE
  8  2,7  - ID
  8  9,5  | IERNE
  8  8,7  | INS
  8  9,5  | IRENE
  8  6,5  - KEN
  8  6,5  - KIN
  8  8,5  | NARES
  8  8,5  | NARIS
  8  5,2  - NEAR
  8  8,5  | NEARS
  8  5,2  - NEER
  8  5,2  - NERE
  8  5,2  - NERI
  8  4,9  | NON
  8  8,5  | RANIS
  8  1,11 - REA
  8  6,4  - SAN
  8  6,4  - SARIN
  8  6,4  - SEN
  8  6,4  - SERIN
  8  6,4  - SIENA
  8  6,4  - SIN
  8  6,4  - SIREN
  8  6,4  - SNARE
  7  5,2  - AERIE
  7  9,5  | AERIE
  7  4,5  | ALE
  7  5,1  | ANA
  7  4,1  | ANC
  7  8,6  | ANES
  7  8,6  | ANIS
  7  9,6  | ANRE
  7  8,5  | ARIES
  7  4,5  | ILA
  7  4,5  | ILE
  7  4,1  | INC
  7  8,8  | ISNT
  7  6,5  - KAE
  7  6,5  - KAI
  7  6,5  - KEA
  7  6,5  - KER
  7  6,5  - KIE
  7  6,5  - KRA
  7  5,6  | MAN
  7  5,6  | MEN
  7  5,6  | MIN
  7  5,6  | MNA
  7 11,8  | NAIG
  7  8,6  | NAIS
  7 10,7  | NAW
  7  5,2  - NEA
  7  5,2  - NEE
  7  5,2  - NEI
  7 10,7  | NEW
  7  8,6  | NIAS
  7  8,6  | NIES
  7 11,8  | RANG
  7  8,7  | RESAT
  7  8,7  | RESIT
  7  4,5  | RLE
  7  6,4  - SAIN
  7  6,4  - SEAN
  7  6,4  - SERAI
  6  3,11 - AA
  6  9,6  | AINE
  6  8,6  | AIRS
  6  7,1  | AN
  6  3,11 - AR
  6  4,1  | ARC
  6 11,5  | ARENA
  6  8,6  | ARES
  6  9,6  | ARNE
  6  3,7  - DA
  6  3,7  - DI
  6  8,6  | EARS
  6  7,1  | EN
  6  8,6  | ERAS
  6  8,6  | ERIS
  6  9,6  | ERNE
  6  7,1  | IN
  6 11,5  | IRENA
  6  8,6  | IRES
  6  5,4  - ISN
  6  6,5  - KA
  6  6,5  - KI
  6  4,6  | LA
  6  4,6  | LI
  6  5,6  | MAE
  6  5,6  | MAR
  6  5,6  | MEA
  6  5,6  | MER
  6  5,6  | MIA
  6  5,6  | MIR
  6  5,2  | NA
  6  5,1  | NAA
  6  9,7  | NAE
  6 11,5  | NAIRA
  6  9,6  | NARE
  6  5,2  - NE
  6  5,1  | NEA
  6  9,7  | NEE
  6  9,8  | NEED
  6  9,8  | NERD
  6  9,6  | NERE
  6  8,7  | NIS
  6  4,9  | NOA
  6  4,9  | NOR
  6  8,6  | RAIS
  6  9,6  | RANE
  6 10,7  | RAW
  6  4,1  | REC
  6 11,5  | REINA
  6  8,6  | REIS
  6  9,8  | REND
  6 10,7  | REW
  6  4,1  | RIC
  6  8,6  | RIES
  6  9,6  | RINE
  6  6,4  - SAIR
  6  6,4  - SARE
  6  6,4  - SARI
  6  6,4  - SEAR
  6  6,4  - SERA
  6  6,4  - SERI
  6  6,4  - SIER
  6  6,4  - SIRE
  5  2,8  - AE
  5  5,2  - AER
  5  8,7  | AES
  5  2,8  | AES
  5  9,6  | AIRE
  5  8,7  | AIS
  5  9,7  | ANE
  5 11,9  | ARG
  5 11,6  | ARNA
  5  8,7  | ARS
  5 10,8  | AW
  5  6,2  - EARN
  5  9,6  | EIRE
  5  9,9  | END
  5  5,1  | ERA
  5  9,6  | ERIE
  5  6,2  - ERIN
  5  6,2  - ERNE
  5  8,7  | ERS
  5 10,8  | EW
  5  9,6  | IARE
  5  5,1  | IRA
  5  8,7  | IRS
  5  5,4  - ISE
  5  3,3  | LI
  5  5,6  | MA
  5  5,6  | ME
  5  5,6  | MI
  5  7,1  | NE
  5  1,3  | OAR
  5  1,3  | OER
  5 11,6  | RANA
  5  8,7  | RAS
  5  2,8  - RE
  5  5,1  | REA
  5  9,8  | READ
  5  9,8  | REED
  5  9,8  | REID
  5  8,7  | RES
  5  2,8  | RES
  5  6,4  - SAE
  5  6,4  - SAI
  5  6,4  - SEA
  5  6,4  - SEI
  5  6,4  - SER
  5  2,4  - SI
  5  6,4  - SIA
  5  6,4  - SIE
  5  6,4  - SIR
  5  6,4  - SRI
  4  5,2  | AA
  4  5,3  | AA
  4  7,1  | AE
  4  5,2  - AE
  4  5,3  | AE
  4  5,3  | AI
  4 11,6  | AIRA
  4 11,7  | ANA
  4  7,7  | ANE
  4  7,1  | AR
  4  9,7  | ARE
  4 11,6  | AREA
  4 11,6  | ARIA
  4  5,4  - AS
  4  8,8  | AS
  4  6,2  - EEN
  4  6,2  - EIRE
  4  7,1  | ER
  4  9,7  | ERE
  4  6,2  - ERIA
  4 11,6  | ERIA
  4  6,2  - ERIE
  4  6,2  - ERN
  4  1,4  - ES
  4  5,4  - ES
  4  8,8  | ES
  4  9,7  | IRE
  4  1,4  - IS
  4  5,4  - IS
  4  8,8  | IS
  4 11,7  | NAA
  4  7,7  | NAE
  4 11,7  | NEA
  4  7,7  | NEE
  4  1,3  | OE
  4  1,3  | OI
  4 11,6  | RAIA
  4  7,1  | RE
  4  9,7  | REE
  4  9,7  | RIE
  3  4,5  | AL
  3  7,7  | ARE
  3  6,2  - EAR
  3  6,2  - EER
  3  6,2  - EIR
  3  4,5  | EL
  3  6,2  - EN
  3  6,2  - ERA
  3 11,7  | ERA
  3  6,2  - ERE
  3  7,7  | ERE
  3 11,7  | IRA
  3  7,7  | IRE
  3 11,8  | NA
  3  7,8  | NE
  3  9,8  | NE
  3 11,7  | REA
  3  7,7  | REE
  3 11,7  | RIA
  3  7,7  | RIE
  3  8,9  | SAT
  3  8,9  | SET
  3  8,9  | SIT
  2 11,8  | AA
  2  7,8  | AE
  2  9,8  | AE
  2  6,2  - EE
  2  7,8  | EE
  2  9,8  | EE
  2  6,2  - ER
  2  7,8  | RE
  2  9,8  | RE
  2  6,4  - SI
