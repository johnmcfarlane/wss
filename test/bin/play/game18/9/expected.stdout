 38  8,4  | ZANIES
 37  8,4  | ZAIRES
 34  6,4  - SIZAR
 34  6,4  - SIZER
 33  6,4  - SIZE
 32  8,4  | XENIAS
 26  6,2  - EXINE
 26  3,1  | NIL
 26  6,4  - SAX
 26  6,4  - SEX
 26  6,4  - SIX
 25  9,5  | RAZEE
 25  9,5  | ZAIRE
 25  5,2  - ZEIN
 24  5,2  - ZEN
 23  3,1  | AIL
 23  8,5  | ZAXES
 23  5,2  - ZEE
 23  7,8  | ZERO
 22  9,7  | ZEE
 20  4,5  | ALIENOR
 20  8,4  | AZINES
 19  1,7  - REDIA
 19  8,5  | ZEINS
 18  8,8  - ANE
 18  7,1  | AX
 18  7,1  | EX
 18  1,7  - ZED
 18  8,6  | ZINS
 17  9,5  | AZINE
 17  8,5  | NAZIS
 17  9,6  | RAZEED
 17  8,5  | ZINES
 16 10,6  | ANEW
 16  8,8  - ARE
 16  8,8  - ERA
 16  8,8  - IRE
 16  8,5  | IZARS
 16  1,11 - NEAR
 16  8,5  | RAZES
 15  7,8  - AIN
 15  9,5  | EXINE
 15  2,6  | FANES
 15  2,6  | FINES
 15  7,8  - NAE
 15  8,5  | NIXES
 15  7,8  - RAN
 15  7,8  - RIN
 15  4,1  | ZAC
 15  9,6  | ZINE
 14  7,8  | AERO
 14  5,5  - AKIN
 14  4,3  | CARLE
 14  6,5  - KEX
 14  9,8  - RAN
 14  8,5  | RAXES
 14  9,6  | RAZE
 14  8,7  | ZAS
 13  7,8  - AIR
 13  8,6  | AXES
 13  8,6  | AXIS
 13  7,8  - EAR
 13  2,6  | FARES
 13  2,6  | FERES
 13  2,6  | FIRES
 13  5,6  | MAX
 13  5,6  | MIX
 13  9,6  | NIXE
 13  7,8  - RIA
 13 11,5  | XENIA
 12  8,8  - AN
 12  3,7  - DIE
 12  8,8  - EN
 12  8,8  - IN
 12  7,8  - NA
 12  7,8  - NE
 12  7,8  - NI
 12  1,7  | RAJ
 12  8,7  | XES
 12  8,7  | XIS
 12  7,7  | ZEE
 11 10,8  - AN
 11  8,10 - AN
 11  9,7  | AXE
 11 10,8  - EN
 11  8,10 - EN
 11  8,10 - IN
 11  6,5  - KAIN
 11  6,5  - KARN
 11  6,5  - KERN
 11  6,5  - KIRN
 11  9,8  - NA
 11  9,8  - NE
 11 11,8  | ZA
 10  7,8  - AE
 10  8,8  - AE
 10  7,4  | AI
 10  7,8  - AI
 10  8,5  | AIRNS
 10  8,8  - AR
 10  7,7  | AXE
 10  4,3  | CARL
 10  4,3  | CEIL
 10  2,11 - EAR
 10  8,5  | EARNS
 10  8,8  - ER
 10  8,7  | INSET
 10  6,5  - KANE
 10  6,5  - KINA
 10  6,5  - KINE
 10  6,5  - KNAR
 10  1,7  - NED
 10  4,1  | NICE
 10  8,5  | RAINS
 10  7,8  - RE
 10  8,5  | REINS
 10  6,4  - SANER
  9  9,8  - AE
  9  1,7  - AID
  9  8,6  | AINS
  9  8,6  | ERNS
  9  6,2  - EX
  9  6,5  - KEIR
  9  6,5  - KIER
  9  5,2  | NAE
  9  9,6  | NEREID
  9  4,1  | RACE
  9  1,7  - RAD
  9  9,5  | RANEE
  9  9,8  - RE
  9  1,7  - RED
  9  4,1  | RICE
  9  1,7  - RID
  9  8,6  | RINS
  9  6,4  - SANE
  9  6,4  - SINE
  8  4,2  | ACE
  8  2,7  - AD
  8  9,5  | ARENE
  8  2,7  - ED
  8  8,7  | ENS
  8  2,6  | FA
  8  2,6  | FE
  8  2,6  | FI
  8  4,2  | ICE
  8  2,7  - ID
  8  8,7  | INS
  8  6,5  - KEN
  8  6,5  - KIN
  8  8,5  | NARES
  8  8,5  | NARIS
  8  5,2  - NEAR
  8  8,5  | NEARS
  8  8,5  | RANIS
  8  6,4  - SAN
  8  6,4  - SARIN
  8  6,4  - SEN
  8  6,4  - SERIN
  8  6,4  - SIN
  8  6,4  - SIREN
  8  6,4  - SNARE
  7  5,2  - AERIE
  7  9,5  | AERIE
  7  4,5  | ALE
  7  5,1  | ANA
  7  8,6  | ANES
  7  8,6  | ANIS
  7  6,5  - KAE
  7  6,5  - KAI
  7  6,5  - KEA
  7  6,5  - KIR
  7  5,6  | MAN
  7  5,6  | MEN
  7 10,7  | NAW
  7  5,2  - NEE
  7 10,7  | NEW
  7  5,2  | RAI
  7 11,8  | RANG
  7  8,7  | RESAT
  7  8,7  | RESIT
  7  6,4  - SAIN
  7  6,4  - SERAI
  6  8,6  | AIRS
  6  7,1  | AN
  6  3,11 - AR
  6  4,1  | ARC
  6 11,5  | ARENA
  6  8,6  | ARES
  6  3,7  - DA
  6  3,7  - DI
  6  8,6  | EARS
  6  7,1  | EN
  6  8,6  | ERAS
  6  9,6  | ERNE
  6  7,1  | IN
  6  8,6  | IRES
  6  6,5  - KA
  6  6,5  - KI
  6  4,6  | LA
  6  4,6  | LI
  6  5,6  | MAE
  6  5,6  | MAR
  6  5,6  | MIR
  6  5,2  | NA
  6  9,7  | NAE
  6 11,5  | NAIRA
  6  5,2  - NE
  6  9,7  | NEE
  6  9,8  | NEED
  6  9,8  | NERD
  6  4,9  | NOR
  6  8,6  | RAIS
  6 10,7  | RAW
  6  4,1  | REC
  6  8,6  | REIS
  6  9,8  | REND
  6  8,6  | RIAS
  6  6,4  - SARI
  6  6,4  - SEAR
  6  6,4  - SERA
  6  6,4  - SIRE
  5  8,7  | AIS
  5  9,7  | ANE
  5  8,7  | ARS
  5 10,8  | AW
  5  6,2  - EARN
  5  9,9  | END
  5  5,1  | ERA
  5  6,2  - ERNE
  5  8,7  | ERS
  5 10,8  | EW
  5  3,3  | LA
  5  3,3  | LI
  5  5,6  | MA
  5  5,6  | ME
  5  5,6  | MI
  5  7,1  | NE
  5  1,3  | OAR
  5  8,7  | RAS
  5  2,8  - RE
  5  9,8  | READ
  5  9,8  | REED
  5  8,7  | RES
  5  2,8  | RES
  5  2,4  - SA
  5  6,4  - SAE
  5  6,4  - SEA
  5  6,4  - SEI
  5  6,4  - SER
  5  2,4  - SI
  5  6,4  - SIR
  5  6,4  - SRI
  4  5,2  | AA
  4  5,3  | AA
  4  7,1  | AE
  4  5,2  - AE
  4  5,3  | AE
  4  5,3  | AI
  4 11,7  | ANA
  4  7,7  | ANE
  4  7,1  | AR
  4  9,7  | ARE
  4 11,6  | AREA
  4 11,6  | ARIA
  4  5,4  - AS
  4  8,8  | AS
  4  6,2  - EEN
  4  7,1  | ER
  4  9,7  | ERE
  4  6,2  - ERN
  4  1,4  - ES
  4  5,4  - ES
  4  8,8  | ES
  4  9,7  | IRE
  4  1,4  - IS
  4  5,4  - IS
  4  8,8  | IS
  4  7,7  | NAE
  4  7,7  | NEE
  4  1,3  | OE
  4  1,3  | OI
  4 11,6  | RAIA
  4  7,1  | RE
  4  9,7  | REE
  3  4,5  | AL
  3  7,7  | ARE
  3  6,2  - EAR
  3  4,5  | EL
  3  6,2  - EN
  3  6,2  - ERA
  3 11,7  | ERA
  3  6,2  - ERE
  3  7,7  | ERE
  3  7,7  | IRE
  3 11,8  | NA
  3  7,8  | NE
  3  9,8  | NE
  3  7,7  | REE
  3 11,7  | RIA
  3  8,9  | SAT
  3  8,9  | SET
  3  8,9  | SIT
  2 11,8  | AA
  2  7,8  | AE
  2  9,8  | AE
  2  6,2  - EE
  2  7,8  | EE
  2  9,8  | EE
  2  6,2  - ER
  2  7,8  | RE
  2  9,8  | RE
  2  6,4  - SA
  2  6,4  - SI
